airflow/contrib/operators/__init__.py

```diff
@@ -174,6 +174,14 @@
             "airflow.providers.google.cloud.operators.dataflow.DataflowTemplatedJobStartOperator"
         ),
     },
+    "datapipeline_operator": {
+        "CreateDataPipelineOperator": (
+            "airflow.providers.google.cloud.operators.datapipeline.CreateDataPipelineOperator"
+        ),
+        "RunDataPipelineOperator": (
+            "airflow.providers.google.cloud.operators.datapipeline.RunDataPipelineOperator"
+        ),
+    },
```

Comment:
Why did you add these?

Reply:
Well, this is the list of deprecated classes, and I added here those that I have deprecated